src/main/java/db/PieceDao.java

```diff
@@ -0,0 +1,55 @@
+package db;
+
+import domain.dto.PieceDto;
+
+import java.util.List;
+
+public class PieceDao {
+    private static final String TABLE_NAME = "pieces";
+
+    private final JdbcTemplate jdbcTemplate;
+    private final RowMapper<PieceDto> rowMapper = (resultSet) -> new PieceDto(
+            resultSet.getString("board_file"),
+            resultSet.getString("board_rank"),
+            resultSet.getString("color"),
+            resultSet.getString("type")
+    );
+
+    PieceDao() {
+        this(new JdbcTemplate());
+    }
+
+    private PieceDao(final JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    void add(final PieceDto piece) {
+        final String query = "INSERT INTO " + TABLE_NAME + " VALUES(?, ?, ?, ?)";
+        jdbcTemplate.add(query, piece.boardFile(), piece.boardRank(), piece.color(), piece.type());
+    }
+
+    PieceDto findOne(final String file, final String rank) {
+        final String query = "SELECT * FROM " + TABLE_NAME + " WHERE board_file = ? and board_rank = ? limit 1";
+        final List<PieceDto> pieces = jdbcTemplate.find(query, rowMapper, file, rank);
+        if (pieces.size() == 0) {
```

Comment:
```suggestion
        if (pieces.isEmpty()) {
```
Java 에서 제공하는 기능을 적극 활용해보죠!

Reply:
반영했어요!